Gemfile

```diff
@@ -44,4 +44,7 @@ group :development, :test do
 end
 
 # Use Rails 12factor
-gem 'rails_12factor', group: :production
\ No newline at end of file
+gem 'rails_12factor', group: :production
+
+# Use foundation-rails
+gem 'foundation-rails'
```

Comment:
All files should end with a newline. Crossed out circle icon indicates that is not the case

http://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline